src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -1,14 +1,16 @@
-import { Button, Divider, Paper, Typography } from "@mui/material";
+import { Button, Divider, Paper, Typography, Box } from "@mui/material";
 import FaultEventShapeToolPane from "./FaultEventShapeToolPane";
 import { EventType, FaultEvent } from "../../../../../models/eventModel";
 import * as React from "react";
 import FailureModeDialog from "../../../../dialog/failureMode/create/FailureModeDialog";
-import { useState } from "react";
-import { EventFailureModeProvider, useEventFailureMode } from "../../../../../hooks/useEventFailureMode";
+import { useState, useEffect } from "react";
+import { EventFailureModeProvider } from "../../../../../hooks/useEventFailureMode";
 import EventFailureModeList from "../failureMode/EventFailureModeList";
 import { FailureMode } from "../../../../../models/failureModeModel";
 import FailureModeShowDialog from "../../../../dialog/failureMode/show/FailureModeShowDialog";
-import { Box } from "@mui/material";
+import useStyles from "@components/editor/faultTree/menu/faultEvent/FaultEventMenu.styles";
+import { asArray } from "@utils/treeUtils";
```

Comment:
This is not treeUtils anymore, I suggest to move it different utils as it is not related to tree at all.

Reply:
Will be resolved https://github.com/kbss-cvut/fta-fmea-ui/issues/262